src/Build/Logging/FancyLogger/FancyLogger.cs

```diff
@@ -43,12 +42,12 @@ public void Initialize(IEventSource eventSource)
             // Started
             eventSource.BuildStarted += new BuildStartedEventHandler(eventSource_BuildStarted);
             eventSource.ProjectStarted += new ProjectStartedEventHandler(eventSource_ProjectStarted);
-            // eventSource.TargetStarted += new TargetStartedEventHandler(eventSource_TargetStarted);
-            // eventSource.TaskStarted += new TaskStartedEventHandler(eventSource_TaskStarted);
+            eventSource.TargetStarted += new TargetStartedEventHandler(eventSource_TargetStarted);
+            eventSource.TaskStarted += new TaskStartedEventHandler(eventSource_TaskStarted);
```

Comment:
I thought we'd decided we weren't showing anything at task granularity; did I misremember?

Reply:
It is used also for updating the spinner. The names of the tasks can be shown inline (eg, Target : Task), but I will make it so that it can be easily hidden in case we don't like it